ci-operator/step-registry/ipi/conf/ibmcloud/sharednetwork/ipi-conf-ibmcloud-sharednetwork-chain.yaml

```diff
@@ -0,0 +1,13 @@
+chain:
+  as: ipi-conf-ibmcloud-sharednetwork
```

Comment:
Try our best to keep consistent format like other platform, not introduce a  dedicated shared network chain yaml file.